src/main/java/com/aws/greengrass/clientdevices/auth/configuration/CAConfiguration.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.configuration;
+
+import com.aws.greengrass.config.Topic;
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.util.Coerce;
+
+import java.util.List;
+
+import static com.aws.greengrass.clientdevices.auth.ClientDevicesAuthService.CA_CERTIFICATE_URI;
+import static com.aws.greengrass.clientdevices.auth.ClientDevicesAuthService.CA_PASSPHRASE;
+import static com.aws.greengrass.clientdevices.auth.ClientDevicesAuthService.CA_PRIVATE_KEY_URI;
+import static com.aws.greengrass.clientdevices.auth.ClientDevicesAuthService.CA_TYPE_TOPIC;
+import static com.aws.greengrass.clientdevices.auth.ClientDevicesAuthService.CERTIFICATE_AUTHORITY_TOPIC;
+import static com.aws.greengrass.componentmanager.KernelConfigResolver.CONFIGURATION_CONFIG_KEY;
+import static com.aws.greengrass.lifecyclemanager.GreengrassService.RUNTIME_STORE_NAMESPACE_TOPIC;
+
+public class CAConfiguration {
+    private final Topics certificateAuthorityTopics;
```

Comment:
Now take this one step further and make this immutable. No need to store topics. We want `CertificateManager` and `CertificateStore` to always be operating on the same config. You can't achieve that if this class is dynamically reading from config